src/gui/TrackContentObjectView.cpp

```diff
@@ -592,28 +613,39 @@ void TrackContentObjectView::mousePressEvent( QMouseEvent * me )
 				gui->songEditor()->m_editor->selectAllTcos( false );
 				m_tco->addJournalCheckPoint();
 
-				// move or resize
+				// Move, Resize, ResizeLeft and Split
 				m_tco->setJournalling( false );
```

Comment:
```suggestion
				// Move, Resize and ResizeLeft
				if (m_action == Move || m_action == Resize || m_action == ResizeLeft)
				{
					m_tco->setJournalling( false );
				}
```

I apologize for another change request, because I missed it myself on the diff that I sent you. It doesn't make much sense for us to disable the TCO journalling just to re-enable it immediately before splitting the TCO. So instead we could just disable it for the actions that require it. `if (m_action != Split)` would work too, but I think checking for the actions that need disabling is better in case we add more actions in the future, so we don't forget about this conditional and it end up disabling it for them too.